### ISSUE TITLE
PMI1 ExternalBarrier broken ("@param center must be an algebra::Vector3D object") with latest IMP/PMI

### ISSUE THREAD

**Issue by benmwebb** [salilab/pmi#178]
Recent changes to ExternalBarrier break the glycophorin PMI1 example:

```
$ git clone https://github.com/integrativemodeling/glycophorin.git
...
$ cd glycophorin/
$ python glycophorin_modeling.py
...
ResidueAngleRestraint: adding a restraint between chainB_99_helix chainB_100_bead_floppy_body chainB_101_bead_floppy_body
Traceback (most recent call last):
  File "glycophorin_modeling.py", line 88, in <module>
    eb = IMP.pmi.restraints.basic.ExternalBarrier(r,50)
  File "/clarinet1/home/ben/imp/release/lib/IMP/pmi/restraints/basic.py", line 54, in __init__
    raise Exception("ExternalBarrier: @param center must be an algebra::Vector3D object")
Exception: ExternalBarrier: @param center must be an algebra::Vector3D object
```

**Comment by cgreenberg** [comment]
yes that should be fixed from this morning's commit: 39d6fbfd544d688b98e24efe51c6f20d255b8fc8

**Comment by cgreenberg** [comment]
confirmed fixed
